boards/arm/stm32_min_dev/doc/stm32_min_dev.rst

```diff
@@ -0,0 +1,73 @@
+.. _stm32_min_dev:
+
+STM32 Minimum Development Board
+###############################
+
+Overview
+********
+
+The STM32 Minimum Development Board, is a popular, cheap breadboard friendly
+breakout board for the STM32F103x8 CPU. Zephyr applications use the
+stm32_min_dev board configuration to run on these boards.
+
+.. figure:: img/stm32_min_dev.jpg
+     :width: 500px
+     :align: center
+     :height: 350px
+     :alt: STM32 Minimum Development Board
+
+     STM32 Minimum Development Board
+
+As the name suggests, these boards have the bare minimum components required to
+power on the CPU. Everything else, you are expected to bug circuit on a
```

Comment:
replace the sentence starting with "Everything else..."  with "For practical use, you'll need to add additional components and circuits using a breadboard, for example."